src/entities/ActiveITSDBClient.ts

```diff
@@ -0,0 +1,163 @@
+import { ConnectionPool, config } from "mssql";
+export default class ActiveITSDBClient {
+  private static instance: ActiveITSDBClient | null = null;
+  private pool: ConnectionPool | null = null;
+
+  dbConfig: config = {
+    user: "readonlyuser",
+    password: "Caltrans2018",
+    server: "10.28.27.238",
+    // server: "sv03tmcswridb",
+    database: "SGDB",
+    options: {
+      encrypt: true, // Enable encryption
+      trustServerCertificate: true, // Bypass certificate validation
+    },
+  };
+
+  private constructor() {}
+
+  static getInstance(): ActiveITSDBClient {
+    if (!ActiveITSDBClient.instance) {
+      ActiveITSDBClient.instance = new ActiveITSDBClient();
+    }
+    return ActiveITSDBClient.instance;
+  }
+
+  initialize(): void {
```

Comment:
As said above, maybe this logic should be done within the `getInstance` function, because I don't think we want multiple DB clients each with their own connection pools?